scripts/zones/Upper_Jeuno/npcs/Mapitoto.lua

```diff
@@ -11,40 +11,19 @@ require("scripts/globals/npc_util")
-    elseif npcUtil.tradeHasExactly(trade, 10062) then player:startEvent(10227, 10062, tpz.ki.TRAINERS_WHISTLE, tpz.mount.WARMACHINE - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.WARMACHINE_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10063) then player:startEvent(10227, 10063, tpz.ki.TRAINERS_WHISTLE, tpz.mount.XZOMIT - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.XZOMIT_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10064) then player:startEvent(10227, 10064, tpz.ki.TRAINERS_WHISTLE, tpz.mount.HIPPOGRYPH - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.HIPPOGRYPH_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10065) then player:startEvent(10227, 10065, tpz.ki.TRAINERS_WHISTLE, tpz.mount.SPECTRAL_CHAIR - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.SPECTRAL_CHAIR_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10066) then player:startEvent(10227, 10066, tpz.ki.TRAINERS_WHISTLE, tpz.mount.SPHEROID - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.SPHEROID_COMPANION)
-
-    -- Omega has own quest and cs (10229)
-    -- elseif npcUtil.tradeHasExactly(trade, 10067) then player:startEvent(10227, 10067, tpz.ki.TRAINERS_WHISTLE, tpz.mount.OMEGA - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.OMEGA_COMPANION)
-
-    elseif npcUtil.tradeHasExactly(trade, 10068) then player:startEvent(10227, 10068, tpz.ki.TRAINERS_WHISTLE, tpz.mount.COEURL - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.COEURL_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10069) then player:startEvent(10227, 10069, tpz.ki.TRAINERS_WHISTLE, tpz.mount.GOOBBUE - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.GOOBBUE_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10070) then player:startEvent(10227, 10070, tpz.ki.TRAINERS_WHISTLE, tpz.mount.RAAZ - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.RAAZ_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10071) then player:startEvent(10227, 10071, tpz.ki.TRAINERS_WHISTLE, tpz.mount.LEVITUS - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.LEVITUS_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10072) then player:startEvent(10227, 10072, tpz.ki.TRAINERS_WHISTLE, tpz.mount.ADAMANTOISE - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.ADAMANTOISE_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10073) then player:startEvent(10227, 10073, tpz.ki.TRAINERS_WHISTLE, tpz.mount.DHAMEL - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.DHAMEL_COMPANION)
-    elseif npcUtil.tradeHasExactly(trade, 10074) then player:startEvent(10227, 10074, tpz.ki.TRAINERS_WHISTLE, tpz.mount.DOLL - 1) player:setCharVar("FullSpeedAheadReward", tpz.ki.DOLL_COMPANION)
+function onTrade(player,npc,trade)    
+    if npcUtil.tradeHasExactly(trade, 10057) or npcUtil.tradeHasExactly(trade, 10067) then
+        -- The Fenrir trade crashes in some arcane away and omega has it's own questline, so bail out here.
```

Comment:
`not` is a thing in lua. Instead of having an empty conditional, just add "not these things" to whatever else is being done and proceed as normal.